wasm-test/src/lib.rs

```diff
@@ -102,3 +104,18 @@ zome_functions! {
         TweetResponse { first: author,  second: content}
     }
 }
+
+#[derive(Serialize, Deserialize)]
+struct TestEntryType {
+    stuff: String,
+}
+
+validations! {
+    [ENTRY] validate_testEntryType {
+        [hdk::ValidationPackage::Entry]
+        |entry: TestEntryType, _ctx: hdk::ValidationData| {
+            (entry.stuff != "FAIL")
+                .ok_or_else(|| "FAIL content is not allowed".to_string())
+        }
```

Comment:
Not blocking but this is overcomplex code logic.
I read this as "if its not invalid return ok, else return error"
Would be simpler to have it as "if invalid return error"

How would this work if you had another invalidity to check for?